packages/zero-solid/src/use-query.test.tsx

```diff
@@ -194,6 +198,401 @@ test('useQuery query deps change', async () => {
   expect(resultDetailsLog).toEqual([{type: 'complete'}]);
 });
 
+test('useQuery query deps change, reconcile minimizes reactive updates', async () => {
```

Comment:
This and the next test cases are new and verify this is giving us the precise reactive updates we want.